rust/osm2lanes/src/transform/tags_to_lanes.rs

```diff
@@ -500,7 +514,11 @@ fn bicycle(
             CYCLEWAY + locale.driving_side.opposite().tag(),
             &["opposite_lane", "opposite_track"],
         ) {
-            return Err("cycleway:BACKWARD=opposite_lane unsupported".into());
+            return Err(RoadMsg::Unsupported {
+                description: None,
+                tags: Some(tags.subset(&[CYCLEWAY + locale.driving_side.opposite().tag()])),
```

Comment:
Cool -- good demonstration of the nice APIs you've been building up, all in one place

Reply:
yup, I am mostly pleased with how this ended up,
especially since I went in without much in the way of a plan :P